examples/hello/bpReader/helloBPReaderHeatMap2D.py

```diff
@@ -41,9 +41,8 @@
 adios = adios2.ADIOS(comm)
 ioWrite = adios.DeclareIO("ioWriter")
 
-varTemperature = ioWrite.DefineVariable("temperature2D", shape,
-                                        start, count, adios2.ConstantDims,
-                                        temperatures)
+varTemperature = ioWrite.DefineVariable("temperature2D", temperatures, shape,
+                                        start, count, adios2.ConstantDims)
```

Comment:
Python code in adios2 uses the autopep8 format https://pypi.org/project/autopep8/0.8/  you might run into problems with this. Usage: `autopep8 -i file.py` for in place formatting.

Reply:
Thanks. I ran autopep8 on it, it's actually happy with it as-is. (Ie, I got lucky)